dbms/src/test/java/org/polypheny/db/cypher/DmlUpdateTest.java

```diff
@@ -106,15 +106,15 @@ public void updatePropertyReturnTest() {
 
 
     @Test
-    @Ignore
+    @Disabled
```

Comment:
Can you enable this test?

Reply:
https://github.com/polypheny/Polypheny-DB/pull/464#discussion_r1520461669